### ISSUE TITLE
Ignoring drag events on nested draggables

### ISSUE THREAD

**Issue by neithere** [jbaysolutions/vue-grid-layout#43]
Hi Gustavo,

Thank you for this awesome library! We've recently switched to it from Gridstack and it has made our code much more elegant.

However, we have stumbled upon an unexpected problem: it's impossible to use nested sortables inside grid elements. It's a blocker for our app.

I've looked inside the code of vue-grid-layout and it seems that the relevant bit of interact.js configuration is hard-coded here:
```js
this.interactObj = interact(this.$refs.item, {ignoreFrom: "a, button"});
```
May I suggest to move this into a property? For example, `GridItem.vue` would look like this:
```js
...
props: {
  ...
  dragIgnoreFrom: {
    type: String,
    required: false,
    default: 'a, button'
  }
},
watch: {
    ...
    draggable () {
      ...
      this.interactObj = interact(this.$refs.item, {ignoreFrom: this.dragIgnoreFrom})
      ...
    },
    ...
}
...
```
The usage pattern would then be:
```html
<grid-item v-for="..." drag-ignore-from=".my-inner-sortable" ... />
```
I tested it locally and it works as a charm. I can make a pull request if you wish.

Cheers,
Andy

**Comment by gmsa** [comment]
Hi Andy, 
A PR would be great!

**Comment by neithere** [comment]
Thank you for merging the PR; any idea when this will be released?

**Comment by gmsa** [comment]
I'll try to make a release tonight«

**Comment by neithere** [comment]
Cool, thank you!

**Comment by gmsa** [comment]
Released 2.1.6 with your changes

**Comment by neithere** [comment]
Fantastic! Thank you!

On 6 Apr 2017 11:13 a.m., "Gustavo Santos" <notifications@github.com> wrote:

Released 2.1.6 with your changes

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/jbaysolutions/vue-grid-layout/issues/43#issuecomment-292115385>,
or mute the thread
